src/main/java/com/deusgmbh/xcusatio/ui/dashboard/ExcusePane.java

```diff
@@ -0,0 +1,115 @@
+package com.deusgmbh.xcusatio.ui.dashboard;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+
+/**
+ * 
+ * This is the ExcusePane class for the User Interface. This class is settled on
+ * the Dashboard and is responsible for the output of the excuses.
+ * 
+ * @author Pascal.Schroeder@de.ibm.com
+ *
+ */
+
+public class ExcusePane extends VBox {
+    private static final String DEFAULT_OUTPUT = "Hier k�nnte Ihre Ausrede stehen";
+    private static final String EXCUSE_TITLE = "Ausrede";
+    private static final int EXCUSE_TITLE_LABEL_FONT_SIZE = 40;
+    private static final int EXCUSE_LABEL_FONT_SIZE = 24;
+
+    private static final double EXCUSE_OPTIONS_PANE_HEIGHT_MULTIPLIER = 0.1;
+    private static final double FEEDBACK_PANE_HEIGHT_MULTIPLIER = 0.25;
+
+    private static final String THUMB_GESTURE_PATH = "file:src/main/resources/thumbGesture.png";
+    private static final int THUMB_GESTURE_HEIGHT = 300;
+
+    private String output;
+    private int thumbGestureValue;
+    private VBox excuseLabelPane;
+    private HBox excuseOptionsPane;
+    private HBox feedbackPane;
+    private BorderPane thumbGesturePane;
+
+    public ExcusePane() {
+        this(DEFAULT_OUTPUT);
+    }
+
+    public ExcusePane(String excuse) {
+        this.output = excuse;
+        this.setAlignment(Pos.CENTER);
+
+        initExcuseLabelPane();
+        initExcuseOptionsPane();
+        initFeedbackPane();
+
+        this.getChildren().addAll(excuseLabelPane, excuseOptionsPane, feedbackPane);
+    }
+
+    public ExcusePane(int thumbGestureValue) {
```

Comment:
So wie ich deinen Code interpretieren würde, handelt es sich bei dem ExcusePane eher um ein "ReactionPane" richtig? Das ExcusePane wäre meiner Ansicht nach eher das Pane, dass das ExcuseLabel-, ExcuseOptions- und FeedbackPane enthält.

Reply:
Ich würde daher vorschlagen diese Klasse umzubennen und eine neue ExcusePane klasse mit den eben genannten Panes zu erstellen und auch das ThumbGesturePane wie die anderen in eine eigene Klasse zu verschieben